lab-abigail/test/test.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const expect = require('chai').expect;
+const textFile = require('../lib/hex.js');
+
+describe('index.js', function() {
+  describe('#Check quality and order of text file output', function() {
```

Comment:
Conventions say that a ``#`` is used to denote a function, i.e. if you were invoking a function called ``sayHello``, your describe block would look something like:

```
describe('testing #sayHello...)
```
So no need to put a ``#`` on Line 7 when you're writing an explicit sentence.